packages/stores/package.json

```diff
@@ -4,6 +4,7 @@
     "version": "1.0.0",
     "main": "src/index.ts",
     "dependencies": {
+        "@deriv/api": "1.0.0",
```

Comment:
also for this we need to update root level `package-lock` using `npm run bootstrap:dev`